CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/#semantic-versioning-200).
 
+## [1.1.12] - 2023-12-21
+### Added
+* Performance test workflow ([PR #517](https://github.com/awslabs/aws-mysql-jdbc/pull/517)).
+* [**Experimental** Enhanced Host Monitoring Plugin v2](https://github.com/awslabs/aws-mysql-jdbc/blob/main/README.md#experimental-enhanced-failure-monitoring-plugin-v2) , which is a redesign of the original Enhanced Host Monitoring Plugin that addresses memory leaks and high CPU usage during monitoring sessions ([PR #513](https://github.com/awslabs/aws-mysql-jdbc/pull/513)).
```

Comment:
Enhanced Host Monitoring Plugin that addresses ->
Enhanced Host Monitoring Plugin and addresses
?

Reply:
Not sure if we need to add performance test workflow in the changelog

Reply:
Done!

Reply:
Didn't see your message @karenc-bq . 
I've now removed performance test workflow.